use-cases/using-SPDK-vhostuser-and-kata.md

```diff
@@ -0,0 +1,120 @@
+# Setup to run SPDK vhost-user devices with Kata Containers and Docker*
+
+- [SPDK Vhost-user target overview](#spdk-vhost-user-target-overview)
+- [Install and setup SPDK vhost-user target](#install-and-setup-spdk-vhost-user-target)
+- [Host setup for vhost-user devices](#host-setup-for-vhost-user-devices)
+- [Launch a Kata container with SPDK vhost-user-blk device](#launch-a-kata-container-with-spdk-vhost-user-blk-device)
+
+## SPDK Vhost-user Target Overview
+
+The Storage Performance Development Kit (SPDK) provides a set of tools and libraries for writing high performance, scalable, user-mode storage applications.
```

Comment:
could you set a maximum of 80 characteres per line?

Reply:
Ack.

Reply:
Thanks for your review, and detailed comments, @devimc